modules/network-graph-20-view.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_content-type: CONCEPT
+[id="network-graph-20-view_{context}"]
+= About the network graph (2.0 preview)
+
+[role="_abstract"]
+
+The network graph provides high-level and detailed information about deployments, network flows, and network policies in your environment.
+
+:FeatureName: Network graph 2.0
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title-short} processes all network policies in each secured cluster to show you which deployments can contact each other and which can reach external networks. It also monitors running deployments and tracks traffic between them. You can view the following items in the network graph:
+
+* Network components: From the top menu, you can select namespaces (indicated by the *NS* label) and deployments (indicated by the *D* label) to display on the graph for a chosen cluster (indicated by the *CL* label). You can further filter deployments by using the drop-down list and selecting criteria on which to filter such as common vulnerabilities and exposures (CVEs), labels, and images.
+* External entities: These represent entities that are connected outside of your cluster. More information is provided in one of the following sections.
+* Network policies: You can view existing policies for a selected component or view components that have no policies.
+* Network flows: You can select one of the following flows for the graph:
+** *Active traffic*: Selecting this default option shows observed traffic, focused on the namespace or specific deployment that you selected. You can select the time period for which to display information.
+** *Inactive flows*: Selecting this option shows potential flows allowed by your network policies, helping you identify missing network policies needed to achieve tighter isolation. You can select the time period for which to display information.
+
+You can also simulate network policies from the network graph view. See "Simulating network policies from the network graph" for version 2.0 or 2.0 preview for more information.
+
+[discrete]
+== Navigation and user interface in the network graph (2.0 preview)
+
```

Comment:
After talking with Mike more, I decided since this content parallels/is based on the Network Graph 1.0 docs, to do a reorganization after the 1.0 docs are removed, to better modularize the content. Version 1.0 is expected to be removed in the next one to two releases.